modules/authentication/snip-enabling-user-authentication-with-github-common-first-steps.adoc

```diff
@@ -0,0 +1,128 @@
+:_mod-docs-content-type: SNIPPET
+.Procedure
+. Allow {product-short} to authenticate with GitHub, by creating a GitHub App.
++
+Use a GitHub App instead of an OAuth app to use fine-grained permissions, use short-lived tokens, scale with the number of installations by avoiding rate limits, and have a more transparent integration by avoiding to request user input.
```

Comment:
are these recommendations/best practices? if yes, I'd put them in an admonition (note)